android/modules/ui/src/java/ti/modules/titanium/ui/widget/picker/TiUINativePicker.java

```diff
@@ -41,7 +41,8 @@
 {
 	private static final String TAG = "TiUINativePicker";
 	private boolean firstSelectedFired = false;
-	private static int defaultTextColor = 0;
+	private static int defaultTextColor = -1;
+	private static boolean setDefaultColor = false;
```

Comment:
If you mean set it to `defaultTextColor` to 0 and using your snippet then it behaves weird initially as text is not visible. Also, this if block would get called x2 for every item with text color set to 0. Seems inefficient?